tools/clang/test/CodeGenHLSL/batch/declarations/resources/norm_floats/assign_different_error.hlsl

```diff
@@ -0,0 +1,14 @@
+// RUN: %dxc -T vs_6_0 -E main %s | FileCheck %s | XFail GitHub #2106
```

Comment:
This should be a syntactic test rather than codegen, but I believe we don't support XFail there.